Automattic/wp-calypso
# Login: Use regular notices rather than global ones for errors on the login page

This pull request fixes an issue noticed in #14090 by changing the general error messages to appear at the top of the login form, rather than using the global notices pattern:

<img width="606" alt="screen shot 2017-05-16 at 11 23 15" src="https://cloud.githubusercontent.com/assets/275961/26101739/1faa8f72-3a2a-11e7-9db2-89d58554028d.png">
  
#### Testing instructions
  
1. Run `git checkout update/login-notice` and start your server, or open a [live branch](https://delphin.live/?branch=update/login-notice)
2. Create a WPCC account from akismet.com, but don't activate it
2. Open the [`Login` page](http://calypso.localhost:3000/)
3. Try to log in
4. Check that you see the error message as above.
5. Log in with a 2FA account
6. Select the SMS option and check that you see a success message above the form
7. Select the SMS option again and check that you see an error message above the form
  
#### Reviews
